docs/analyst_guide.md

```diff
@@ -0,0 +1,217 @@
+# Access levels
+
+Users can have one of the following access levels to a database:
+
+- `direct` - Direct (non-anonymized) access to data. Restrictions listed in this document do not apply in direct mode.
+- `anonymized_trusted` - Anonymized access to data. Prevents accidental release of personal data.
+- `anonymized_untrusted` - Anonymized access to data. Prevents intentional release of personal data.
+
+Use `SELECT diffix.access_level()` to see the current access level.
+
+# Anonymized queries
+
+In anonymized access level, queries targeting personal tables are restricted to a limited subset of SQL.
+A personal table is a relation that contains data of individuals or other protected entities.
+Administrators identify and mark such tables during configuration.
+
+When selecting data from personal tables, the following queries are allowed.
+
+## Queries with grouping
+
+Grouping queries have the following form:
+
+```
+SELECT col1, col2, ..., count(...)
+FROM personal_table
+GROUP BY col1, col2, ...
+```
+
+Where `col1`, `col2`, ... `colX` are table columns.
+Numeric and string columns can optionally be generalized.
+
+`count()` is any of the supported [count aggregate](#count) variants.
+
+**Example:**
+
+```
+SELECT city, last_name, count(*)
```

Comment:
`last_name` not a good example column because normally it would be heavily suppressed. Maybe `city, year_of_birth, count(*)`? Or something